src/doctor.rs

```diff
@@ -166,3 +166,17 @@ pub fn check_seaorm_cli() -> Check {
         },
     }
 }
+
+/// Prints the configuration and its path.
+pub fn print_config(config: &Config) -> Result<(), Error> {
```

Comment:
This function is missing tests.
 It would be better to modify it so that it doesn't return a string directly, allowing us to use the `insta` crate to test the output. 
Alternatively, we could implement `fmt::Display` for the struct to handle printing the config details.

Reply:
Thank you for the feedback! I have implemented `fmt::Display` for the `Config` struct.

Reply:
can you add test?

Reply:
Thank you for your patience. I have added the test.